config/config.go

```diff
@@ -180,7 +180,7 @@ func GetTxnScopeFromConfig() string {
 
 // ParsePath parses this path.
 // Path example: tikv://etcd-node1:port,etcd-node2:port?cluster=1&disableGC=false
-func ParsePath(path string) (etcdAddrs []string, disableGC bool, err error) {
+func ParsePath(path string) (etcdAddrs []string, disableGC bool, keyspace string, err error) {
```

Comment:
All variable names and function names should clearly indicate whether they are ```keyspaceName``` or ```keyspaceID```, and do not use only ```keyspace```.
Otherwise, the code maintenance cost is very high

Reply:
Then it should be named by the query convention: `keyspace_name`?